### PR TITLE
test(lottie): slot regression test

### PR DESCRIPTION
**Pull request by theashraf**

### CODE REVIEW

File: test/testLottie.cpp [hunk 1]
```diff
@@ -262,6 +262,79 @@ TEST_CASE("Lottie Quality", "[tvgLottie]")
...
+        REQUIRE(canvas->draw() == Result::Success);
+        REQUIRE(canvas->sync() == Result::Success);
+
+        REQUIRE(animation->apply(0) == Result::Success);
+
+        REQUIRE(canvas->update() == Result::Success);
+        REQUIRE(canvas->draw() == Result::Success);
+        REQUIRE(canvas->sync() == Result::Success);
+
+        REQUIRE(picture->load(TEST_DIR"/slot.json") == Result::Success);
```
Comment (tinyjin) [inline test/testLottie.cpp]: Unit test is failing becuase `load` is called after rendering fired. Once the `draw()`/`update()` is called, it would raise `InsufficientCondition`.

Could you separate test cases, or define different animations for this?

**Review by tinyjin — CHANGES_REQUESTED** [review]
Thanks. please check comments

File: test/testLottie.cpp (hunk 1 above)
Comment (tinyjin) [reply test/testLottie.cpp]: Slot API usage looks correct, just usage of `load` is invalid.

Comment (theashraf) [reply test/testLottie.cpp]: @tinyjin I've removed the invalid usage of `load`

**Comment by hermet** [comment]
> Image
> I am wondering if it's the case reproduces edge case we've faced. When I check out to v1.0.0 (which had issue), and run this unit test, it passes.

@tinyjin Did you try with the sanitizer? `-Db_sanitize="address,undefined"`

**Comment by tinyjin** [comment]
@hermet I tested both, same passes with `Lottie Slot Regression Test` TC

**Comment by hermet** [comment]
@tinyjin If you've confirmed, we don't necessarily need to apply. 
@theashraf Thanks for your contribution!
